internal/pkg/db/db.go

```diff
@@ -39,17 +41,22 @@ type (
 	Option func(*Backend)
 )
 
-func NewBackend(opts ...Option) *Backend {
+func NewBackend(opts *options.Options) (*Backend, error) {
 	backend := &Backend{
 		Backend: ent.NewBackend(),
 		Logger:  utils.NewLogger("db"),
 	}
 
-	for _, opt := range opts {
-		opt(backend)
+	backend.
+		Debug(opts.Debug).
```

Comment:
Need to make sure verbosity is 2 or higher to enable SQL statement debugging for the ent client

Reply:
Not sure of what action is needed here

Reply:
Using `-v` enables debug logging, using `-vv` or higher should enable the debug config on the ent client. That's the check for `verbosity >= minDebugLevel` you've probably seen in a few places

Reply:
So should we be checking the verbosity level in this file? I can add it to options.Options and do the comparison in `WithDebug` or maybe just pass the verbosity into that function?